BookOfHabits/BookOfHabitsMicroservice.Domain.ValueObjects/RoomName.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace BookOfHabitsMicroservice.Domain.ValueObjects
+{
+    public class RoomName
+    {
+        public string Name { get; }
+        public RoomName(string name)
+        {
```

Comment:
As a rule, the validation of fields should be executed in validators in order to not allow processing invalid data in API.